src/main/java/frc/robot/Config.java

```diff
@@ -237,4 +241,51 @@ public static final class AutoConstants {
         new TrapezoidProfile.Constraints(
             kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
   }
+  public class ArmConfig {
+    public static final boolean SET_INVERTED = true;
+
+    public static final int CURRENT_LIMIT = 60;
+
+    //soft limit constant for bottom arm
+    public static final float arm_forward_limit = (float)Math.toRadians(135);
+    public static final float arm_reverse_limit = (float)Math.toRadians(40);
+    public static final boolean SOFT_LIMIT_ENABLE = true;
+    
+    //PID constants
+    public static final double arm_kP = 1.4;
+    public static final double arm_kI = 0.0003;
+    public static final double arm_kD = 0.9;
+    public static final double arm_kIz = 0.3;
+    public static final double arm_kFF = 0;
+    public static final double min_output = -1;
+    public static final double max_output = 1;
+
+    //ff calculations 
+    public static final double gravitationalConstant = 389.0886; //inches/s/s which is equal to 9.81 m/s/s
+    public static final double ARM_FORCE = 11.29 *gravitationalConstant; //11.29 lb
+
+    public static final double LENGTH_ARM_TO_COG = 14.56;
+
+    //arm offsets
+    //public static final double bottom_arm_offset = 307.800000;
+    //public static final double top_arm_offset = 0;
+
+    //syncing encoders
+    public static double ENCODER_SYNCING_TOLERANCE = 0.01; //radians
+
+
+    public static final double ARM_NEO_GEAR_RATIO =  1;
```

Comment:
This should be renamed to `ARM_ENCODER_GEAR_RATIO`

The neo gear ratio might be 1:60 but the encoder gear ratio will be 1:1